py2app/recipes/autopackages.py

```diff
@@ -1,10 +1,19 @@
 import os
 
 AUTO_PACKAGES=[
-    # Embbedded datafiles accessed using 
+    # Embbedded datafiles accessed using
     # ``__file__`` relative paths.
```

Comment:
FYI: I filed #368 to look into autodetecting the usage of ``__file__``, that would take away the need for a significant portion of the items in autopackages.

Reply:
Packages I included here experience `[Errno 20] Not a directory: ` errors due to some use of `os` path functions. What I've encountered so far are the use of `os.listdir` (in the case of `debugpy`) and `os.exists`, both of which result in the error if the directory or file is located in the zip folder.